<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>add_goods</class>
 <widget class="QDialog" name="add_goods">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>729</width>
    <height>117</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add goods</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="goodsNameLabel">
           <property name="text">
            <string>Назва ТМЦ</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="goodsNameLine"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="goodsUnitsLabel">
           <property name="text">
            <string>Одиниці виміру</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="goodsUnitsLine"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="goodsPriceLabel">
           <property name="text">
            <string>Ціна</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="goodsPriceLine"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="cancel">
         <property name="text">
          <string>Відмінити</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="save">
         <property name="text">
          <string>Зберегти</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
